### ISSUE TITLE
how to install

### ISSUE THREAD

**Issue by wolnavi** [isohuntto/openbay#111]
I am using byet host and i unzipped and uploaded the open bay master folder to the ftp via file zilla. but i don't know how to continue after step 4 (https://github.com/isohuntto/openbay/wiki/shared-hosting-guide). i visited the website but it doesnt show any installer

**Comment by nexusrain** [comment]
Why arent you taking a look at my guide.. Its now updated and also explains, how to host the database on your own server. See here: https://www.bitforce.io/hacking/howto-setup-openbay-on-debian-ubuntu-with-nginx-1-6-2/

**Comment by anomp** [comment]
I have a similar problem so I do not open a new issue.I have an ubuntu 12.04 server.When I copy zip files under /var/www directory and trying to open the Panel from browser,it opens like ftp directory.I think I have my apache misconfigured but I do not know how to make it right.Any help?

**Comment by AlanGreyjoy** [comment]
I'll screen shot how it should look. I'm on the toilet right now lolololol.

**Comment by HiddenKn** [comment]
@AlanGreyjoy Still on the toilet? Must be having some problems of your own then. :laughing:
